baker_wet_cleaning_application/scripts/map_handling_behavior.py

```diff
@@ -23,6 +19,7 @@ def setParameters(self, database_handler, rooms_list):
 		# Parameters set from the outside
 		self.database_handler_ = database_handler
 		self.rooms_list_ = rooms_list
+		self.is_finished = False
```

Comment:
why needed?

Reply:
For continuing path exploration, which was interrupted by dirt detections or similar.